src/Core/src/Hosting/Internal/AppHost.cs

```diff
@@ -6,21 +6,21 @@
 
 namespace Microsoft.Maui.Hosting.Internal
 {
-	internal class AppHost : IAppHost, IAsyncDisposable
+	class AppHost : IAppHost, IAsyncDisposable
 	{
 		readonly ILogger<AppHost>? _logger;
 
 		public AppHost(IServiceProvider services, ILogger<AppHost>? logger)
 		{
 			Services = services ?? throw new ArgumentNullException(nameof(services));
-			Handlers = Services.GetRequiredService<IMauiHandlersServiceProvider>();
+			Handlers = Services.GetService<IMauiHandlersServiceProvider>();
```

Comment:
why are we changing this? i think we really should crash if there's no `IMauiHandlersServiceProvider`

Reply:
Yup.